### PR TITLE
Democratize Releases 

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

For too long only the few elite bourgeoisie were given the agency to publish modules to NPM. Today, my friends, I say no more! With this PR we, the proletariat, take back the power (to release to npm)! Do you need a module released? Don't ask, just open a PR. No one needs to merge directly to main anymore.

Here's the new process:
1. Update the versions of the components via `npm run version`
2. Update change logs to have the current date for the release of the affected components.
3. Create a Release PR, Get approvals
4. Merge PR to `main`
5. Watch main to ensure releases get pushed to npm and tagged in github appropriately

Closes #

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Testing

I tested all this releasing over here on a fork:
https://github.com/mjhenkes/terra-framework
https://travis-ci.com/github/mjhenkes/terra-framework/builds/176004718

### Additional Details

The revolution has begun.

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: scripts/release/gitTagMonoRepo.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
```
Comment (mjhenkes) [inline scripts/release/gitTagMonoRepo.js:1]: it would be great if we had a scripts package to put this in for all repos..

**Comment by emilyrohrbough** [comment]
@mjhenkes How do we ensure the release date is the PR merge date? We often have old PRs that get merged in and that date could be hard to manage.

**Comment by mjhenkes** [comment]
@emilyrohrbough, you know what my answer will be, get better at merging PRs, Ideally release PRs will be simple, just a change logs and version updates so they'll be quick.

And on the other hand.... how accurate to the dates really need to be? I wager they'll be close enough and if you want a super specific date, check the tag.

**Comment by emilyrohrbough** [comment]
Any way to automate the release version in the release script you wrote up?

**Comment by mjhenkes** [comment]
Yes it could be done, but we can do that as part of a future release.

**Comment by mjhenkes** [comment]
Another not here is that, after this pr gets merged, I'd like at least one successful release of terra-framework before applying this to terra-core or terra-clinical

**Pull request merged**